revolv/templates/project/project.html

```diff
@@ -262,21 +384,59 @@ <h4 class="title-h4">{{ donor.get_full_name }}</h4>
+                                <!-- end .row -->
+                                <!--  </nav> -->
+                                <!-- end .nav -->
+                                <!--       <div class="tab-content"><div class="container"><div class="tab-content-section active"><div class="row">
+              
+              <div class="col-md-6"><div class="info-box"><h4 class="title-h4">Joe3 Donor3</h4></div></div>
+              
+              <div class="col-md-6"><div class="info-box"><h4 class="title-h4">admin</h4></div></div>
+              
+              <div class="col-md-6"><div class="info-box"><h4 class="title-h4">parul raheja</h4></div></div>
+               -->
+                                <!-- end .col-md-6 -->
+                                <!--     </div></div> -->
+                                <!-- end .tab-gold -->
+                                <!--         </div></div> -->
+                                <!-- end .tab-content -->
+                                <!--     </div> -->
+                                <!-- end .tabs -->
+                            </div>
+                                <!-- end .project-updates-module -->
+                                <!-- <div class="donors-module"><div class="mains-tabs"><nav class="tab-index"><div class="container"><div class="row"><div class="col-lg-3 col-md-3 col-sm-6"><a href="javascript:;" class="active">DONORS</a></div></div> -->
+                                <!--  </div> -->
+                                <!-- end .row -->
+                                <!--  </nav> -->
+                                <!-- end .nav -->
+                                <!--       <div class="tab-content"><div class="container"><div class="tab-content-section active"><div class="row">
+              {% for donor in project.donors.all %}
+              <div class="col-md-6"><div class="info-box"><h4 class="title-h4">{{ donor.get_full_name }}</h4></div></div>
+              {% endfor %} -->
+                                <!-- end .col-md-6 -->
+                                <!--     </div></div> -->
+                                <!-- end .tab-gold -->
+                                <!--         </div></div> -->
+                                <!-- end .tab-content -->
+                                <!--     </div> -->
+                                <!-- end .tabs -->
+                            </div>
+                            <!-- end .donors-module -->
+                        </div>
+                        <!-- end .contents -->
 {% endblock %}
-
-
 {% block javascripts %}
-<script type="text/javascript"
-    src="//maps.googleapis.com/maps/api/js?key={{GOOGLEMAPS_API_KEY}}">
-</script>
 
-<script src="https://checkout.stripe.com/checkout.js"></script>
 
-<script>
+                        <!-- tab start here -->
+                        <!-- closed her3 -->
+                        <script type="text/javascript"
```

Comment:
The formatting changes of this `script` tag do not need to happen, please revert them to how they were before.